flow/Makefile

```diff
@@ -396,11 +396,11 @@ $(RESULTS_DIR)/1_1_yosys.v: $(DONT_USE_LIBS) $(WRAPPED_LIBS) $(DONT_USE_SC_LIB)
 
 $(RESULTS_DIR)/1_synth.v: $(RESULTS_DIR)/1_1_yosys.v
 	mkdir -p $(RESULTS_DIR) $(LOG_DIR) $(REPORTS_DIR)
-	cp $< $@
+	ln $< $@
```

Comment:
What happened to the -s?

Reply:
Full explanation here: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1000#issuecomment-1515814679